### ISSUE TITLE
Batch render to local directory and move files to destination afterwards

### ISSUE THREAD

**Issue by tappi287** [tappi287/pfadaeffchen#1]
On very instable networks, the maya batch renderer will fail with the following error:
```
gui_create_process: Warning: line 0: The post-processing failed while attempting to rename file //192.168.13.xxx/xxx_pfad.iff to //192.168.13.xxx/render_output/20180713_1531490110_4359/xxx_pfad.iff.ng.
gui_create_process: The operating system reported this error: "Permission denied".
```
An enhancement should be:
* point the maya batch renderer to a stable local directory
* let the image watcher process move the files to the destination in multiple retries

**Comment by tappi287** [comment]
This error message results from a bug when rendering is set to software. Image processing thread accessed img data of batch renderer. Changing image watcher behaviour.

**Comment by tappi287** [comment]
fixed with v0.96.6
